internal/backends/sqlite/metadata/registry.go

```diff
@@ -352,6 +354,27 @@ func (r *Registry) CollectionRename(ctx context.Context, dbName, oldCollectionNa
 	panic("not implemented")
 }
 
+// Version returns SQLite version.
+//
+// If there are no databases in the registry, empty string is returned.
+func (r *Registry) Version(ctx context.Context) (string, error) {
+	if r.version != "" {
+		return r.version, nil
+	}
+
+	db := r.p.GetFirst(ctx)
+	if db == nil {
+		return "", nil
+	}
+
+	row := db.QueryRowContext(context.Background(), "SELECT sqlite_version()")
```

Comment:
Let's use `ctx` passed from the function parameter :hugs:

Reply:
fixed